hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/CreateOperationTracking.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.evm.contracts.operations;
+
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+
+public class CreateOperationTracking {
+    protected boolean failForExistingHollowAccount(MessageFrame frame, Address contractAddress) {
```

Comment:
I thinks this functionality should exist inside `hedera-evm` in the `AbstractEvmRecordingCreateOperation` and we can remove this method from CreateOperationTracking